src/app/common/error/ErrorModal.tsx

```diff
@@ -63,7 +68,7 @@ export default class ErrorModal extends PureComponent<ErrorModalProps> {
 
         return (
             <Fragment>
-                { message && <p>{ message }</p> }
+                { message && <p id="errorMessage">{ message }</p> }
```

Comment:
I'd give it a more specific ID, e.g.: `errorModalMessage`. The reason is because ID has to be unique for the whole document. So `errorMessage` could potentially clash with other error messages in the document, e.g.: alert messages, error messages for a form etc...

Reply:
updated, thanks!